assets/javascripts/kitten/components/lines/info-lines/index.js

```diff
@@ -0,0 +1,130 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import { pxToRem } from '../../../helpers/utils/typography'
+import { ScreenConfig } from '../../../constants/screen-config'
+
+const StyledInfoLines = styled.div`
+  ${({ borderColor }) =>
+    borderColor &&
+    css`
+      border-top: ${pxToRem(1)} solid ${borderColor};
+      border-bottom: ${pxToRem(1)} solid ${borderColor};
+    `}
+
+  ${({ withBorderRadius }) =>
+    withBorderRadius &&
+    css`
+      border-radius: ${pxToRem(8)};
+    `}
+
+  ${({ withLeftRightBorder, borderColor }) =>
+    withLeftRightBorder &&
+    borderColor &&
+    css`
+      border-left: ${pxToRem(1)} solid ${borderColor};
+      border-right: ${pxToRem(1)} solid ${borderColor};
+    `}
+
+  ${({ withoutTopBottomBorder }) =>
+    withoutTopBottomBorder &&
+    css`
+      border-top: none;
+      border-bottom: none;
+    `}
+
+  ${({ withoutResponsive }) =>
+    withoutResponsive &&
+    css`
+      ${StyledLine} {
+        flex-direction: row;
+      }
+      ${StyledKey} {
+        flex: auto;
+      }
+    `}
+`
+
+const StyledLine = styled.div`
+  flex-direction: column;
+  display: flex;
+  padding: ${pxToRem(15)};
+
+  ${({ borderColor }) =>
+    borderColor &&
+    css`
+      :not(:last-child) {
+        border-bottom: ${pxToRem(1)} solid ${borderColor};
+      }
+    `}
+
+  @media (min-width: ${ScreenConfig.M.min}px) {
+    flex-direction: unset;
```

Comment:
Le `unset` n'est pas accepté par IE11, autant définir complètement :
 
```suggestion
    flex-direction: row;
```